charts/atlantis/values.yaml

```diff
@@ -228,6 +228,9 @@ hideUnchangedPlanComments: false
 defaultTFVersion: ""
 # Example: "0.12.0".
 
+### -- Sets which TF distribution to use. Can be set to terraform or opentofu.
```

Comment:
And since it needs to match the readme file:
```suggestion
### -- Sets the default terraform distribution to use. Can be set to terraform or opentofu.
```